plugins/woocommerce/src/Internal/ProductQueryFilters/FilterClausesGenerator.php

```diff
@@ -1,254 +1,202 @@
 <?php
-namespace Automattic\WooCommerce\Blocks;
+/**
+ * FilterClausesGenerator class file.
+ */
+
+namespace Automattic\WooCommerce\Internal\ProductQueryFilters;
 
 use WC_Tax;
 use Automattic\WooCommerce\Internal\ProductAttributesLookup\LookupDataStore;
 
+defined( 'ABSPATH' ) || exit;
+
 /**
- * Process the query data for filtering purposes.
+ * Class for filter clauses.
  */
-final class QueryFilters {
```

Comment:
If this was ever released, then it's not backward compatible. You would have already considered but just checking so that we keep this in mind.

Reply:
> If this was ever released, then it's not backward compatible. 

You're right. Even though we haven't announced it or documented it, it was released publicly. I will follow your suggestion convert it to an empty shell and deprecate it.

Reply:
I addressed this in https://github.com/woocommerce/woocommerce/pull/43772/commits/9126b42ac2e8e26efce1c0cec595d9fe3b47a2bc